test/medium/fb_messenger/fb_messenger_test.exs

```diff
@@ -0,0 +1,14 @@
+defmodule Medium.FbMessengerTest do
+  use ExUnit.Case
```

Comment:
We should use `ExUnit.Case` from now on when testing modules.

Reply:
gotcha